pkgs/tools/audio/opl3bankeditor/common.nix

```diff
@@ -0,0 +1,62 @@
+{ pname, chip, version, sha256 }:
+
+{ mkDerivation
+, stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, qttools
+, qtbase
+, qwt
+, rtaudio
+, rtmidi
+}:
+
+let
+  binname = "${chip} Bank Editor";
+  mainProgram = "${lib.strings.toLower chip}_bank_editor";
+in
+mkDerivation rec {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "Wohlstand";
+    repo = pname;
+    rev = "v${version}";
+    fetchSubmodules = true;
```

Comment:
Would it be a good idea to devendor the submodules?

Reply:
Generally yes, I just didn't want to bother with it this week. I've opened an issue about it - https://github.com/Wohlstand/OPL3BankEditor/issues/185 - but I'll look into patching the CMake script to use our packages instead.